dev/confirm-dialog.html

```diff
@@ -6,8 +6,16 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Confirm dialog</title>
     <script type="module">
+      import '@vaadin/polymer-legacy-adapter/style-modules.js';
+
       import '@vaadin/confirm-dialog';
+      import '@vaadin/vaadin-lumo-styles/color.js';
+      import '@vaadin/vaadin-lumo-styles/typography.js';
     </script>
+    <!-- Global color and typography styles affect the contents of the dialog -->
+    <custom-style>
+      <style include="lumo-color lumo-typography"></style>
+    </custom-style>
```

Comment:
Let's avoid using `<custom-style>`

```suggestion
      import '@vaadin/confirm-dialog';
      import { color, typography } from '@vaadin/vaadin-lumo-styles';

      // Global color and typography styles affect the contents of the dialog
      const style = document.createElement('style');
      style.innerHTML = `${color.toString()} ${typography.toString()}`;
      document.head.appendChild(style);
```